test/RequestStreamTest.cpp

```diff
@@ -49,6 +49,31 @@ TEST(RequestStreamTest, HelloSync) {
   ts->assertValueAt(9, "Hello Bob 10!");
 }
 
+TEST(RequestStreamTest, DISABLED_HelloFlowControl) {
+  folly::ScopedEventBaseThread worker;
+  auto server = makeServer(std::make_shared<TestHandlerSync>());
+  auto client = makeClient(worker.getEventBase(), *server->listeningPort());
+  auto requester = client->getRequester();
+  auto ts = TestSubscriber<std::string>::create(5);
+  requester->requestStream(Payload("Bob"))
+      ->map([](auto p) { return p.moveDataToString(); })
+      ->subscribe(ts);
+
+  // cause data to flow?  I can't use awaitTerminalEvent, it sits forever
```

Comment:
what's 'cause data to flow' refer to?

Reply:
When line 65 executes, nothing has returned yet.  But I don't want to use the testSubscriber's awaitTerminalEvent(), since it will block forever (since I'm not requesting the full 10 all at once and no onComplete will be called)

Reply:
Makes sense. We probably want a method on TestSubscriber that says "awaitUntilNumElements(int n)" and have that block until 1) a timeout of some hundreds of ms expires (and throw), 2) onComplete/onError (and throw), 3) onNext is called 'n' times (and return normally).